packages/bitstringStatusList/src/identifier/bitstring-status-list-entry-store.ts

```diff
@@ -165,22 +169,38 @@ export class BitstringStatusListEntryStore {
         throw new Error('Bitstring Status List not found');
       }
 
-      const vc = JSON.parse(bitstringStatusList.verifiableCredential!);
+      const vc: W3CVerifiableCredential = JSON.parse(
+        bitstringStatusList.verifiableCredential!,
+      );
+      let unsignedVC: UnsignedCredential;
+      let credentialSubject: CredentialSubject;
+      if (typeof vc === 'string') {
+        const decoded = decodeJWT(vc);
+        unsignedVC = decoded?.payload as UnsignedCredential;
+
+        credentialSubject =
+          decoded?.payload?.vc?.credentialSubject ||
+          decoded?.payload?.credentialSubject;
+      } else {
+        credentialSubject = vc.credentialSubject;
+        unsignedVC = { ...vc };
+        delete unsignedVC.proof;
+      }
+      unsignedVC.issuanceDate = new Date().toISOString();
```

Comment:
Should we replace `issuanceDate` with `validFrom` ?

Reply:
Good point! I remove it and update [here](https://github.com/gs-gs/veramo/commit/848a2a1cd2ff6c0548b54da61a532c874d5a9dae)